### PR TITLE
add cron to container

### LINKED ISSUE #65: Authentication tokens are never purged
Authentication tokens expire after 2 hours by default, after which they'll live happily in the database forever, unless I'm missing something.

This might have to wait until there's some kind of cron system in place?

### PR DESCRIPTION
**Pull request by amdwit**
partly fixes #65

### CODE REVIEW

File: app/docker-compose.prod.yml [hunk 1]
```diff
@@ -60,6 +60,8 @@ services:
     networks:
       - media_nginx
       - envoy_nginx
+  cron:
+    build: cron
```
Comment (aapeliv) [inline app/docker-compose.prod.yml:64]: If this is a separate container, it needs to be built in CI/CD, hopefully tested, then the prod docker-compose should pull the image from gitlab, it shouldn't be built in prod.

File: app/cron/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FROM node:14-alpine
```
Comment (aapeliv) [inline app/cron/Dockerfile:1]: Why node?

**Review by aapeliv — CHANGES_REQUESTED** [review]
The larger issue encompassing background jobs is #282. I am not sure if cron is the right solution, since we want certain jobs to run fairly frequently. I think a more full-fledged scheduler which could also basically run async jobs without much delay would be more appropriate. For this, we need some kind of queueing, triggers, etc.

In terms of backend coding, it would be easier to make backend scheduled and async jobs to run within the current backend framework, it's a pain to share code between two directories, as we've found with the media container. The way to do that might be two dockerfiles in the same directory, or the same container, but running as a background job worker with certain config, and otherwise as a API worker.

**Comment by amdwit** [comment]
Maybe another solution is better.

**Pull request closed without merging**